src/orb.yaml

```diff
@@ -0,0 +1,156 @@
+
+examples:
+  waiting_for_containers:
+    description: "Waiting for containers to be ready for start testing"
+    usage:
+      version: 2.1
+      orbs:
+        wait-for: cobli/wait-for@x.x.x
+      jobs:
+        build:
+          docker:
+            - image: circleci/openjdk:8
+            - image: cassandra:3.2
+          steps:
+            - checkout
+            - wait-for/cassandra
+            - run:
+                name: Test
+                command: sbt test
+
+commands:
+  sh-command:
+    description: Waits for a shell command to be sucessfully executed
+    parameters: &wait-for-command-parameters
+      description:
+        description: command name
+        type: string
+        default: command
+      timeout:
+        description: timeout
+        type: integer
+        default: 60
+      seconds_between_retries:
+        description: seconds
+        type: integer
+        default: 1
+      sh_command:
+        description: command to be checked
+        type: string
+        default: "true"
+    steps:
+      - run:
+          name: Wait for <<parameters.description>>
+          command: |
+            timeout <<parameters.timeout>> sh -c \
+              'until <<parameters.sh_command>>; do
+                echo "Waiting for <<parameters.description>> ..."
+                sleep <<parameters.seconds_between_retries>>
+              done'
+
+  port:
+    description: Waits for a port being up
+    parameters: &wait-for-port-parameters
+      <<: *wait-for-command-parameters
+      description:
+        description: port description
+        type: string
+        default: port
+      host: *localhost-parameter
+      port: *port-parameter
+    steps:
+      - sh-command:
+          <<: *wait-for-attributes
+          sh_command: nc -z <<parameters.host>> <<parameters.port>>
+
+  kafka:
+    description: Waits for Kafka being up
+    parameters:
+      <<: *wait-for-command-parameters
+      description:
+        description: kafka description
+        type: string
+        default: Kafka
+      zookeeper_host: *localhost-parameter
+      zookeeper_port:
+        <<: *port-parameter
+        default: 2181
+    steps:
+      - sh-command:
+          <<: *wait-for-attributes
+          sh_command: >-
+            echo dump |
+            nc <<parameters.zookeeper_host>> <<parameters.zookeeper_port>> |
+            grep brokers
+
+  localstack-s3:
+    description: |
+      Waits for a LocalStack container with S3 service being up
+    parameters:
+      <<: *wait-for-port-parameters
+      description:
+        description: LocalStack S3 description
+        type: string
+        default: Localstack S3
+      port:
+        <<: *port-parameter
+        default: 4572
+    steps:
+      - sh-command:
+          <<: *wait-for-attributes
+          sh_command: >-
+            curl --silent --fail
+            "http://<<parameters.host>>:<<parameters.port>>"
+
+  redis:
+    description: Waits for redis to be ready
```

Comment:
Why we change the description from **being up** to **be ready**?

Reply:
Normalized to  *be ready*